connectors/sources/notion.py

```diff
@@ -567,6 +577,21 @@ def generate_query(self):
                     "filter": {"value": "database", "property": "object"},
                 }
 
+    def is_connected_property_block(self, page_database):
+        properties = page_database.get("properties")
+        if properties is None:
+            return False
+        for field in properties.keys():
+            if field.startswith("Related to") and (
+                "(Figma File)" in field
+                or "(Google Drive File)" in field
+                or "(GitHub Pull Requests)" in field
+                or "(Zendesk Ticket)" in field
+            ):
```

Comment:
Do we need this part of the check? Is `field.startswith("Related to")` not sufficient?
What I'm concerned about is that they'll have some other type of integration added for something like OneDrive, and then `Related to (OneDrive)` will not match this expression and we'll see this error again.

Reply:
Or maybe we could make it a regex match for `Related to .* connected app .*`?